storage/persistent_storage/rosa-persistent-storage-aws-ebs.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: ASSEMBLY
+[id="rosa-persistent-storage-aws-ebs"]
+= Persistent storage using AWS Elastic Block Store (EBS)
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-persistent-storage-aws-ebs
+
+toc::[]
+
+{product-title} (ROSA) clusters are pre-built with two storage classes that use AWS Elastic Block Store (EBS) volumes. These storage classes are ready to use and some familiarity with Kubernetes and AWS is assumed.
+
+Following are the two pre-built storage classes:
```

Comment:
```suggestion
Following are the two prebuilt storage classes:
```
This is a sentence fragment, but you could consider just making it a table title (using a `.Heading`) to avoid having to say too much :)

Reply:
Made the change.